base/sim/FairDetector.cxx

```diff
@@ -124,6 +125,22 @@ void FairDetector::Initialize()
     fMC = TVirtualMC::GetMC();
 }
 
+// -------------------------------------------------------------------------
+
+void FairDetector::ProcessHits()
+{
+    static const auto printOnce = [this] {
+        LOG(warning) << "Calling depracated Bool_t FairDetector::ProcessHits(FairVolume* vol = 0) for \"" << GetName()
+                     << "\".";
+        LOG(warning) << "        Replace with void FairDetector::ProcessHits(). ";
```

Comment:
Okay, having looked at it again… Now I understand more!

Maybe rephrase to somethig like "Please override / implement ProcessHits on " << GetName() << " instead of the old ProcessHits(FairVolume*)".